api/candlepin-api-spec.yaml

```diff
@@ -31,6 +30,20 @@ paths:
         default:
           $ref: '#/components/responses/default'
 
+  /openapi.yaml:
```

Comment:
As I described in the other comment, we're already serving this statically, so there is no reason for a new endpoint

Reply:
Removed.